addons/web/static/src/views/list/list_renderer.js

```diff
@@ -2075,6 +2077,25 @@ export class ListRenderer extends Component {
         if (ev.target.closest(".ui-autocomplete")) {
             return;
         }
+        // Save, Discard
+        if (target.closest(".o_control_panel_breadcrumbs")) {
```

Comment:
I would specifically target the buttons, because with the current version, clicking in the blank area next to those buttons does not switch the row back in readonly, but it should.
```suggestion
        if (target.closest(".o_list_button_save") || target.closest(".o_list_button_discard")) {
```